src/Sulu/Bundle/SnippetBundle/Content/SnippetContent.php

```diff
@@ -251,4 +252,23 @@ private function loadSnippets($ids, $webspaceKey, $locale, $shadowLocale = null)
 
         return $snippets;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getReferencedUuids(PropertyInterface $property)
+    {
+        $data = $property->getValue();
+
+        return $this->getUuids($data);
+    }
+
+    /**
+     * The data is not always normalized, so we normalize the data here.
+     */
+    private function getUuids($data)
+    {
+        $ids = is_array($data) && array_key_exists('ids', $data) ? $data['ids'] : array();
+        return $ids;
```

Comment:
I think its a bug in jackalope. Anyway, this is just moving the logic from one place to another, the contents of this is another issue.